test/metric_value_benchmark/agent_configs/statsd_config.json

```diff
@@ -1,22 +1,16 @@
 {
-    "agent": {
-      "metrics_collection_interval": 15,
-      "run_as_user": "root",
-      "debug": true,
-      "logfile": ""
-    },
     "metrics": {
-      "namespace": "MetricValueBenchmarkTest",
-      "append_dimensions": {
-        "InstanceId": "${aws:InstanceId}"
-      },
-      "metrics_collected": {
-        "statsd": {
-            "metrics_aggregation_interval": 15,
-            "metrics_collection_interval": 15,
-            "service_address": ":8125"
-        }
-      },
-      "force_flush_interval": 5
+        "namespace": "statsd_test",
```

Comment:
unimportant, but why are we changing the namespace here?

Reply:
It made it easier for me to find the metrics specific from this test.
It isn't a required change.
I could change it back.

Reply:
I do want to have a collection interval < aggregation interval though.

Reply:
that's fine